docs/system/k8s-metrics.md

```diff
@@ -1089,5 +1091,38 @@ This metric is [recommended][MetricRecommended].
 <!-- END AUTOGENERATED TEXT -->
 <!-- endsemconv -->
 
+## K8s Container metrics
+
+**Description:** K8s Container level metrics captured under the namespace `k8s.container`.
```

Comment:
same as on the other PR, is this statement  necessary?

Reply:
As in https://github.com/open-telemetry/semantic-conventions/pull/2197#discussion_r2113406300, we can remove those all together in separate PR?